Makefile

```diff
@@ -26,11 +26,17 @@ ci: toolchain verify licenses battletest ## Run all steps used by continuous int
 test: ## Run tests
 	ginkgo -r
 
-battletest: ## Run stronger tests
+
+strongertests:
 	# Run randomized, parallelized, racing, code coveraged, tests
 	ginkgo -r \
-		-cover -coverprofile=coverage.out -outputdir=. -coverpkg=./pkg/... \
-		--randomizeAllSpecs --randomizeSuites -race
+			-cover -coverprofile=coverage.out -outputdir=. -coverpkg=./pkg/... \
+			--randomizeAllSpecs --randomizeSuites -race
+
+deflake:
+	for i in {1..10}; do make strongertests; done
+
+battletest: strongertests
 	go tool cover -html coverage.out -o coverage.html
```

Comment:
why did this fail when it wasn't broken out into a separate target?

Reply:
I think when you run it multiple times it corrupts the file or something like that you can see the outcome here https://github.com/aws/karpenter/runs/5504735463?check_suite_focus=true

```
cover: line "" doesn't match expected format: couldn't find a   before Count
make: *** [Makefile:32: battletest] Error 1
Error: Process completed with exit code 2.
```

Regardless we don't need the coverage conversion at every step because it will just add to the total time taken.